test/specs/scenarios/navigation.js

```diff
@@ -0,0 +1,37 @@
+import { describe, it } from '../../utils/mochaw'
+import { goToPassportUploadScreen, uploadFileAndClickConfirmButton } from './sharedFlows.js'
+
+const options = {
+  pageObjects: [
+    'Welcome',
+    'DocumentSelector',
+    'DocumentUpload',
+    'Confirm',
+    'BasePage'
+  ]
+}
+export const navigationScenarios = async(lang) => {
+  describe(`NAVIGATION scenarios in ${lang}`, options, ({driver, pageObjects}) => {
+
+    const { welcome, documentSelector, documentUpload, confirm, basePage } = pageObjects
+    const copy = basePage.copy(lang)
+
+    it('should navigate to the second-last step of the flow and then go back to the beginning', async () => {
+      goToPassportUploadScreen(driver, welcome, documentSelector, `?language=${lang}&async=false&useWebcam=false`)
+      uploadFileAndClickConfirmButton(documentUpload, confirm, 'passport.jpg')
+      documentUpload.getUploadInput()
+      documentUpload.upload('face.jpeg')
+      basePage.clickBackArrow()
```

Comment:
this should be `documentUpload.clickBackArrow()`
@bartzet I thought this had already been changed, no?

Reply:
ah, I see it's being done in this PR: https://github.com/onfido/onfido-sdk-ui/pull/753/files#diff-000f2b48b64835ff8b6ce05ce531a95bL645

So good with me!